connect/cli/plugins/shared/translation_sync.py

```diff
@@ -151,6 +156,7 @@ def _action_create(self, context_id, data):
             },
         }
         translation = self._client.ns('localization').translations.create(payload)
+        self._new_translation = translation
```

Comment:
several translations can be created in a `sync()` call, so only the last created one will be tracked. Is this the intended behaviour?

Reply:
Yes, also right with this, I was assuming that first we a create a translation, then update the attrs for this translation, but the truth is that all the translations are created first, so we need here to have a list to pass it to `TranslationAttributesSynchronizer` at the end of creation.